packages/core-modules/contracts/submodules/election/ElectionTally.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.0;
 import "./ElectionBase.sol";
 
 /// @dev Defines core vote-counting / ballot-processing functionality in ElectionModule.evaluate()
-abstract contract ElectionTally is ElectionBase {
+contract ElectionTally is ElectionBase {
```

Comment:
The generic ElectionModule is no longer abstract. It's just dumb, but it can be deployed.

Reply:
to clarify, in practicality, should it ever be deployed? is it useful for testing, for example? trying to figure out what "dumb" means here :laughing: